spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -22,6 +22,10 @@
       expect(json).to have_key("steam")
       expect(json['steam']).to have_key("url")
       expect(json['steam']).to have_key("nickname")
+      expect(json['bans']['mute']).to eq(false)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.